documentation/starter-kits/README.md

```diff
@@ -46,21 +46,31 @@ To leverage this template, make sure to do the following:
 
 A contribution guide is an important document directed at *new contributors* to your project - to help them in understanding what and how they may contribute. It can be used to outline expected requirements prior to contributions (i.e. acceptance of licensing terms, signing off on any developer agreements required) as well as to ensure the quality of contributions are in-line with expected project norms. If a developer asks how to contribute, this is the guide to point them to, and if patches are submitted to a project but are missing essential ingredients, this is again the guide to point new developers to.
```

Comment:
If a developer asks how to contribute, please point them to this guide.  This guide also itemizes essential ingredients necessary for a successful patch submission.

Reply:
Thanks for the better wording 👍